TACC/Core-Styles
# test(components): c-button, selectors

## Overview

Support `.button`, `.button--1`, et cetera along with original `.c-button`, `.c-button--primary`.

## Related

- preparation for **upcoming task to support light button colors on dark section**

## Changes

- add `c-button.selectors.css`
- use selectors instead of `.c-button(…)`

## Testing

0. Checkout this PR's branch.
1. <details><summary>Review affected patterns…</summary>

    - components
    	- admonition
    	- button
    	- card
    	- form
    	- message
    	- django cms forms
    - elements
    	- links
    - objects
    	- section
    - trumps
    	- links (irregular)

    </details>

3. Verify styles have **not** changed (compared to `main`).

## UI

https://user-images.githubusercontent.com/62723358/222807712-4adb4ff8-9199-4c4f-b799-70ef8a442c58.mov
